cpp/src/interop/to_arrow_schema.cpp

```diff
@@ -102,7 +97,7 @@ int dispatch_to_arrow_type::operator()<numeric::decimal64>(column_view input,
                                                            ArrowSchema* out)
 {
   using DeviceType = int64_t;
-  return decimals_to_arrow<DeviceType>(input, out);
+  return decimals_to_arrow<DeviceType>(input, cudf::detail::max_precision<DeviceType>() - 1, out);
```

Comment:
Why are we subtracting 1 from the precision here?

Reply:
Added a comment to explain the subtraction:

decimal64 max precision is 18 for Arrow, cudf uses `floor(bits * log(2) / log(10))` which comes out to 19, so we need to subtract 1.